src/jsonapi-datastore.js

```diff
@@ -114,7 +114,19 @@ class JsonApiDataStore {
     if (!this.graph[type] || !this.graph[type][id]) return null;
     return this.graph[type][id];
   }
-
+   /**
+    * Retrieve all models by type, attribute and value.
+    * @method findByAttr
+    * @param {string} type The type of the model.
+    * @return {object} The corresponding model if present, and null otherwise.
+    */
+   findByAttr(type, attr, value) {
+     if (!this.graph[type]) return null;
+     var typeDatas = this.graph[type];
```

Comment:
How about using `Array.filter`? Here you don't really retrieve all models matching attribute and value, only the first one.